projects/packages/videopress/src/class-data.php

```diff
@@ -14,6 +14,16 @@
  * The Data class.
  */
 class Data {
+
+	/**
+	 * Gets the Jetpack blog ID
+	 *
+	 * @return int The blog ID
+	 */
+	public static function get_blog_id() {
+		return VideoPressToken::blog_id();
```

Comment:
probably we'd like to add the whole code here instead of calling the token method. Something that we can address (mayyybeeee) in a follow-up one. just a comment.